src/test/cpp/net/socketservertestcase.cpp

```diff
@@ -478,4 +477,4 @@ LOGUNIT_CLASS(SocketServerTestCase)
 const File SocketServerTestCase::TEMP("output/temp");
 const File SocketServerTestCase::FILTERED("output/filtered");
 
-LOGUNIT_TEST_SUITE_REGISTRATION_DISABLED(SocketServerTestCase)
+LOGUNIT_TEST_SUITE_REGISTRATION(SocketServerTestCase)
```

Comment:
Should this test really be enabled? While your CMAKE-build might handle things automatically, it might be more difficult for users without CMAKE. Took me a while to figure things out based on this commit myself and the test still fails currently...

	set PATH=C:\Program Files\Java\jdk-8\bin;C:\Program Files (x86)\UnxUtils\usr\local\wbin;%PATH%
	set TEST_PROJ=%cd%
	set LOG4CXX_VER_BASE=..\..\..\..
	set TEST_SRC=%LOG4CXX_VER_BASE%\src\src\test
	set TEST_RES=%TEST_SRC%\resources

	set TOTO=wonderful
	set key1=value
	set key2=value2

	set log4j_CLASSPATH=%LOG4CXX_VER_BASE%\..\..\..\..\Java\log4j-1.2.17.jar
	set SOCKET_SERVER_SOURCES=%TEST_SRC%\java\org\apache\log4j\net\ShortSocketServer.java
	set SOCKET_SERVER_CLASSPATH=%TEST_RES%;%log4j_CLASSPATH%
	set CLASSPATH=%SOCKET_SERVER_CLASSPATH%
	set SOCKET_SERVER_COMMAND=C:\Program Files\Java\jdk-8\bin\java.exe;org.apache.log4j.net.ShortSocketServer;8;input/socketServer

	pushd "%TEST_RES%"
	javac -d "." -classpath "%log4j_CLASSPATH%" "%SOCKET_SERVER_SOURCES%"
	"%TEST_PROJ%\Win32\Debug\out\%~n0.exe" %*
	popd

	goto :EOF

Reply:
Disabling a testcase is needed when building multiple tests into a single executable. To run a disabled test case you then need to add it as a parameter.
The socketservertestcase excutable has only one test case so I do not think it is appropriate to disable it.

Reply:
The point is not about different executables, but that this one test case only requires additional Java and is difficult to setup. It has been disabled in the past for a long time already:

https://github.com/apache/logging-log4cxx/commit/b279aa7922fe8cbdc3ab2da524e1d6a9ed3913bb#diff-78b9ec5d590a88f742877f2f096c52a9R51